picoscope/darwin_utils.py

```diff
@@ -27,7 +29,13 @@ def LoadLibraryDarwin(library):
             FileNotFoundError
         except NameError:
             FileNotFoundError = IOError
-        if not Path(PICO_LIB_PATH).is_dir():
+
+        PICO_LIB_PATH = None
+        for path in PICO_LIB_PATHS:
+            if Path(path).is_dir():
+                PICO_LIB_PATH = path
```

Comment:
I don’t think checking for the ecistance of the directory is enough. I would like this to run through more of the library loading code. Maybe more the old code into a function and just call it.

Reply:
I guess maybe I would just prefer this test:

```
        for p in PICO_LIB_POSSIBLE_PATHS:
            if (Path(PICO_LIB_PATH) / library).is_file():
                PICO_LIB_PATH = p
                break
        else:
            raise FileNotFoundError(
                "Could not find library {}.".format(library))

```